### PR TITLE
console api since that's exactly what modders need

### PR DESCRIPTION
**Pull request by jearmstrong21**
console api since that's exactly what modders need
console api since that's exactly what modders need
console api since that's exactly what modders need
console api since that's exactly what modders need
console api since that's exactly what modders need

### CODE REVIEW

**Comment by jearmstrong21** [comment]
E

File: src/main/java/io/github/fabriccommunity/everything/PrintAPIBuilder.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package io.github.fabriccommunity.everything;
+
+public class PrintAPIBuilder(){
```
Comment (Juuxel) [inline src/main/java/io/github/fabriccommunity/everything/PrintAPIBuilder.java:3]: Not valid Java.

Comment (jearmstrong21) [reply src/main/java/io/github/fabriccommunity/everything/PrintAPIBuilder.java]: Typo! Should be fixed now.

**Comment by valoeghese** [comment]
haha yes

**Pull request merged**